ARExperiment/ARExperiment/SizeComparisonViewController.swift

```diff
@@ -99,3 +101,14 @@ extension SizeComparisonViewController: ARSCNViewDelegate {
         return node
     }
 }
+
+extension SizeComparisonViewController: ARExperimentSessionHandler {
```

Comment:
this extension is exactly the same everywhere. Could it be factored out ?

Reply:
@malaba yes this was part of a bigger delegate where they were different. But we have done it much simpler now. 

How would you factor it out though?